### DISCUSSION TITLE
Applied Science の訳について

### DISCUSSION THREAD

**Discussion by denev-rove** [nsk4762jp/OxygenNotIncluded-Japanese#429] [General]
現行では、`Applied Science` と `Materials Science` の訳が入り乱れている状態なので、これを整理したいです。

- `Applied Sciences Research` point → `応用科学研究`ポイント
複製人間が`材料工学端末`を使って研究することで獲得できる研究ポイント。
（研究ポイントが溜まるとテクノロジーを獲得できる）
- `Applied Sciences Research` skill → `材料工学研究`スキル
複製人間が習得するスキルの名称。
- `Applied Sciences Booster` → `材料工学ブースター`
バイオニック複製人間が使うブースターの名称
- `Materials Science Research` → `材料工学研究`
研究ポイントを溜めて開発するテクノロジーの名称。
- `Materials Study Terminal` → `材料工学端末`
複製人間が研究を行う設備の名称。

原文での表記揺れは、おそらく開発過程で発生した仕様変更の名残だと予想します。
（ハイパーリンク用のタブ名には `NUCLEAR` とか `ATOMIC` なんて単語も出てきて、さまざまな検討があったことが伺える）

これを整理する場合、
- 原文どおりに`応用科学`と`材料工学`を使い分ける
- `Applied Science` も `Materials Science` も全て`材料工学`で統一する

の2つの方法が考えられると思います。
私としては後者がいい（わざわざ使い分ける利点はない）と思いますが、ご意見ください。

**Comment by nsk4762jp** [comment]
出現頻度で見るとどっちも同じくらいなんですね。
変えるとしたら `Applied Science` も `材料工学` にする、としたほうが影響度も低くていいと思います。
